rep/processors/CtGovPdfProcessor.py

```diff
@@ -44,17 +57,27 @@ def _process_blob_wrapper(self, blob):
             [x[0] for x in vote_list]
         )
 
-
     def _get_page_from_blob(self, blob):
         fileReader = PyPDF2.PdfFileReader(io.BytesIO(blob))
         page_content = fileReader.getPage(0).extractText()
         page_content = page_content.replace('\n', '')
         page_content += '\n'
         return page_content
 
-
-    def _write_to_csv(self, year, date, bill_number, vote_name, rep_name, rep_vote):
+    def _write_to_db(self, year, date, bill_number, vote_name, rep_name, rep_vote):
+        # with open('bills.csv', 'a') as f:
         for num, _ in enumerate(rep_vote):
             logging.info(f"""
```

Comment:
I know I added this log line, but it's really noisy, I think it should either be `logging.debug`, so it doesn't blow up stdout, or it should just get removed entirely

Reply:
I'm fine with removing it, I almost never read the log info because you're right, it's too noisy